tests/test_server.py

```diff
@@ -560,9 +523,8 @@ async def test_clear_environment(client, server, clienthelper, environment):
     )
     assert result.code == 200
 
-    result = await client.get_environment(id=environment, versions=10)
+    result = await client.environment_get(id=environment)
     assert result.code == 200
-    assert len(result.result["environment"]["versions"]) == 1
```

Comment:
I believe that this change and the one below weaken the core of the test case.

Reply:
Added some assertions on the version count